src/components/Footer/footer.tsx

```diff
@@ -0,0 +1,76 @@
+import { Link } from "react-router-dom";
+import logo from "./logo.png";
+import inst from "./instagram.svg";
+import fb from "./facebook.svg";
+import "./footer.css";
+
+const Footer = () => {
+  return (
+    <div className="footer-menu">
+      <div className="footer-logo">
+        <img src={logo} alt="Healthcare logo" width="300" height="150"></img>
+      </div>
+      <ul id="footer-nav">
+        <h1 className="footer-header">Navigation</h1>
+        <li className="footer-nav-item">
+          <a href="/home">Homepage</a>
+        </li>
+        <li className="footer-nav-item">
+          <a href="/aboutUs">About us</a>
+        </li>
+        <li className="footer-nav-item">
+          <a href="/profile">Profile</a>
+        </li>
+        <li className="footer-nav-item">
+          <a href="/services">Services</a>
+        </li>
+      </ul>
+
+      <ul id="footer-rules">
+        <h1 className="footer-header">Rules</h1>
+        <li className="footer-nav-item">
+          <a href="/privacyPolicy">Privacy policy</a>
+        </li>
+        <li className="footer-nav-item">
+          <a href="/terms">Terms of service</a>
+        </li>
+      </ul>
+      <ul id="footer-links">
```

Comment:
You can split this to smaller components for example section nav menu and contacts and import these components to new Page/Footer